acceptance/analyzer_test.go

```diff
@@ -532,13 +564,13 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 										h.WithFlags(append(
 											dockerSocketMount,
 											"--network", registryNetwork,
-											"--env", "CNB_REGISTRY_AUTH="+cacheAuthConfig,
+											"--env", "CNB_REGISTRY_AUTH="+fixtures.authRegAuthConfig,
```

Comment:
Before this change we used the second return value from `buildAuthRegistryImage` (which is the `authConfig` instead of the `regRepoName`).

Reply:
Since authConfig is per registry, not per repo, we only need to set it once :)